app/presenters/api_docs/api_reference.rb

```diff
@@ -3,8 +3,10 @@ class APIReference
     attr_reader :document
     delegate :servers, to: :document
 
-    def initialize(spec)
+    def initialize(spec, version: '1.0')
       @document = Openapi3Parser.load(spec)
+      @version = version
+      @highlight_yaml = @version == '1.0' ? {} : YAML.load_file("config/vendor_api/v#{@version}-highlights.yml")
```

Comment:
This presenter is used across multiple APIs so we should not be specifying the vendor API here, fixing in commit incoming